SPECS/postgresql/postgresql.spec

```diff
@@ -65,12 +71,21 @@ Obsoletes:      libpq-devel < 13
 The postgresql-devel package contains libraries and header files for
 developing applications that use postgresql.
 
+%package	service
+Summary:	To setup postgresql as a service
+Requires:	%{name} = %{version}-%{release}
+Requires:	util-linux
```

Comment:
Is the Requires needed at all times or just for "pre"-installation and "post" installation?
Requires:
    vs
Requires(pre):
Requires(post):